src/js/media-theme-element.ts

```diff
@@ -222,11 +231,11 @@ export class MediaThemeElement extends globalThis.HTMLElement {
   }
 
   createRenderer(): void {
-    if (this.template && this.template !== this.#prevTemplate) {
-      this.#prevTemplate = this.template;
+    if (this.#template && this.#template !== this.#prevTemplate) {
+      this.#prevTemplate = this.#template;
 
       this.renderer = new TemplateInstance(
-        this.template,
+        this.#template,
```

Comment:
this needs to be reverted because it might access 
`(this.constructor as typeof MediaThemeElement).template;`